src/Compilers/CSharp/CscCore/csc

```diff
@@ -2,5 +2,10 @@
 
 THISDIR=$(dirname $0)
 
+# CoreCLR / CoreFX does not execute reliably with an unlimited 
+# stack.  Make on Linux can execute child processes with an 
+# unlimited stack so force it back to a known good limit
+ulimit -s 8192
+
```

Comment:
Note: I will be filing a CoreCLR bug on this once this PR is merged.  I want to have a read made repro for them to look at.  Once I file the bug I'll push a new commit which changes this entry to reference that new bug number.